### ISSUE TITLE
URL Parameter

### ISSUE THREAD

**Issue by NSTA1** [A9T9/Free-OCR-API-CSharp#2]
When I supply the URL property to /Parse/Image, I get this response:  

`{"ParsedResults":null,"OCRExitCode":99,"IsErroredOnProcessing":true,"ErrorMessage":"Not a valid URL. URL must be of a valid and existing Image or PDF.","ErrorDetails":"","ProcessingTimeInMilliseconds":"0"}`

The url provided is a valid one  (an Azure Blob url with a valid SAS token - tested and working), so I'm guessing that the service doesn't like the fact that the url has a query string...  could you please allow query strings for such scenarios?  (Would be nice if my app could just point you at the Azure blob instead of having to download it from Azure in order to upload it to your service)

**Comment by A9T9** [comment]
I agree, that should be working. Can you please send me the image link for
testing?

**Comment by NSTA1** [comment]
Here's a demo image with a SAS token valid for a week.

https://kbw.blob.core.windows.net/test/friday.jpg?sv=2014-02-14&sr=c&sig=XuFb1o%2FyL%2Bxmvf9eMaNmZ4vtCty9RdNcOcjNH0wrUZM%3D&st=2016-03-11T00%3A00%3A00Z&se=2016-03-19T00%3A00%3A00Z&sp=r

**Comment by NSTA1** [comment]
Need a new one? Any update on this? @A9T9

**Comment by A9T9** [comment]
Thanks for asking. No need link required. We confirmed the issue, and will
fix this asap, hopefully by early April the latest.

**Comment by A9T9** [comment]
Well, it is May meanwhile and the issue is not fixed yet. But we are currently preparinga bigger update for June, and it will be included then.
